acurl/pyproject.toml

```diff
@@ -1,2 +1,2 @@
 [build-system]
-requires = ["setuptools", "wheel", "Cython<3.0"]
+requires = ["setuptools", "wheel", "Cython==3.0.8"]
```

Comment:
can we pin to a version range?  pinning to an exact version means we won't pick up future bugfixes...

Reply:
Pinned to `<=3.0.8,>4.0.0`